localMatchApp/logIn.h

```diff
@@ -0,0 +1,17 @@
+#include <string>
+
+
+class logIn
+{
+public:
+  logIn();
+  ~logIn();
+  logIn ( const logIn & ) = default;
+  logIn &  operator= ( const logIn & ) = default;
+
+  int signUp(const std::string i_user, const std::string i_password);
+
+  bool isEqualPass(const std::string i_password, const std::string i_repeatedPassword);
```

Comment:
these param names are not equal as in localMatchApp/logIn.cpp
please change those names as they are in this file

Reply:
Done